cm-server/src/main/java/ro/msg/cm/handler/ServiceResponseExceptionHandler.java

```diff
@@ -0,0 +1,78 @@
+package ro.msg.cm.handler;
+
+import com.google.gson.Gson;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.CannotCreateTransactionException;
+import org.springframework.util.StringUtils;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+import ro.msg.cm.exception.CandidateIsAlreadyValidatedException;
+import ro.msg.cm.exception.CandidateNotFoundException;
+import ro.msg.cm.exception.PatchCandidateInvalidValueException;
+
+import javax.persistence.EntityNotFoundException;
+import javax.validation.ConstraintViolationException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+@ControllerAdvice
+public class ServiceResponseExceptionHandler extends ResponseEntityExceptionHandler {
+    private final static HttpHeaders HEADERS;
+    private final static String MESSAGE = "message";
+
+    static {
+        HEADERS = new HttpHeaders();
+        HEADERS.setContentType(MediaType.APPLICATION_JSON);
+    }
+
+    @ExceptionHandler(value = CandidateNotFoundException.class)
+    protected ResponseEntity<Object> handleCandidateNotFoundException(RuntimeException ex, WebRequest request) {
```

Comment:
Create a POJO with a message field (e.g. `ErrorMessage`). Return `ResponseEntity<ErrorMessage>` instead of `ResponseEntity<Object>`.